extralibs/lib_rand.h

```diff
@@ -0,0 +1,133 @@
+/******************************************************************************
+* Psuedo Random Number Generator using a Linear Feedback Shift Register
+* See the GitHub for more information:
+* https://github.com/ADBeta/CH32V003_lib_rand
+*
+* Ver 1.1    09 Sep 2024
```

Comment:
lib_rand_test.c says "Ver 2.0" - is "Ver 1.1" here correct?

Reply:
Demo is ver2, lib is ver1.1

Reply:
Thanks for the clarification.